sirepo/server.py

```diff
@@ -104,7 +104,7 @@ def api_copySimulation(self):
         )
         return self._save_new_and_reply(req, d)
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec("require_user", filename="FileName", file_type="FileType")
```

Comment:
SimFileName and SimFileType

Reply:
24fd189959a5d6096a9482e74f1ec55c9faeb6de